website/content/docs/configuration/listener/index.mdx

```diff
@@ -9,7 +9,33 @@ description: |-
 # `listener` Stanza
 
 The `listener` stanza configures the addresses and ports on which Boundary will
-respond to requests. At this time, there are two listeners - [TCP][tcp] and [Unix][unix].
+respond to requests. At this time, there are two listener types:
+[TCP](/docs/configuration/listener/tcp) and [Unix](/docs/configuration/listener/unix).
+
+
+## `listener` Purpose
+
+The `purpose` field is used to determine what part of Boundary's functionality is
+started and exposed for a given listener block. Each listener must have a `purpose` set.
+
+Currently, Boundary recognizes the following listener purposes:
+
+- `api`: Starts up and exposes Boundary's controller API.
+  At least one `api` listener must be present if a `cluster`
+  purpose listener is present. By default, it runs on `:9200`.
+
+- `cluster`: Starts up and exposes Boundary's worker and
+  controller communication layer. Only one `cluster` listener
+  is allowed per Boundary instance. Must be present if an `api`
+  purpose listener is present. By default, it runs on `:9201`.
```

Comment:
I think it should be made clear that these are needed for a boundary Controller instance.

Reply:
Addressed in [95f3c08](https://github.com/hashicorp/boundary/pull/1882/commits/95f3c08a0fc1d54b3cb55e1d7dbbfe838d0318b7)